pkg/stanza/fileconsumer/file_sort.go

```diff
@@ -10,7 +10,7 @@ import (
 	"strconv"
 	"time"
 
-	strptime "github.com/observiq/ctimefmt"
```

Comment:
We should definitely rely on our [internal timeutils](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/internal/coreinternal/timeutils) package. There are subtle issues with parsing timestamps that we don't want to solve independently.

Reply:
I believe this was an old commit.